### ISSUE TITLE
Executing multiple statements shouldn't stop on error

### ISSUE THREAD

**Issue by hugo187** [jakob/Postico#174]
It seems when executing multiple statements and one of them encounters an error, the remaining statements are not executed. I believe the default behaviour should be that all statements are executed regardless of any errors. 

That is how psql treats its input by default (could be changed by ON_ERROR_STOP parameter). The same does phpPgAdmin when executing a query. Adminer has an option "Stop on error", which is disabled by default.

I got caught up by this recently when I was running series of database queries which were wrapped in a transaction block (BEGIN; .... COMMIT;). An error in one of the queries stopped execution so the final COMMIT statement was't been reached and the database wasn't rolled back into the pre-transaction state. Furthermore, being in the middle of failed transaction causes any Postico operations to fail until COMMIT is explicitly executed.

**Comment by jakob** [comment]
Postico sends all the queries to the server in a single multi-statement query. Therefore it is not really Postico that decides to abort, but rather the PostgreSQL server.

This behavior is a bit different than psql, but I believe it has a very big advantage: everything you execute is implicitely wrapped in a transaction. So if you execute 3 statements at once, you have a guarantee that either all or none of them will succeed.

Yes, this is annoying when you use explicit transactions, but I believe it is saver overall.

Also, instead of executing COMMIT, it would probably be more apropriate to execute a ROLLBACK, even if the two effectively do the same thing when the transaction is in a failed state.

**Comment by hugo187** [comment]
Thanks Jakob. I didn't know multiple queries are wrapped in a transaction. It makes sense and I actually like it as well.

It's just that it's not really obvious. Perhaps, this could be mentioned somewhere in the manual? E.g. in the "Using the SQL Query view" chapter once it's written?

**Comment by jakob** [comment]
Yes, that's a good point. I really need to start writing this chapter, since the query view does differ from psql in a lot of ways....
